src/main/java/com/timgroup/statsd/NamedPipeSocketAddress.java

```diff
@@ -14,6 +14,16 @@ public String getPipe() {
         return pipe;
     }
 
+    /**
+     * Return true if object is a NamedPipeSocketAddress referring to the same path.
+     */
+    public boolean equals(Object object) {
```

Comment:
Is this used anywhere?

Reply:
It is used implicitly in tests when we `assertEquals` two `NamedPipeSocketAddress`es